src/comm_manager.cpp

```diff
@@ -622,6 +680,23 @@ void CommManager::send_next_param(void)
   }
 }
 
+void CommManager::send_next_config_info(void)
+{
+  if (send_device_info_index_ < Configuration::DEVICE_COUNT)
+  {
+    if (send_config_info_index_ == 0)
+      send_device_info(send_device_info_index_);
+    else
```

Comment:
Alternatively, have
```c++
if (send_config_info_index_ == 0)
  send_device_info(send_device_info_index_

send_config_info(send_device_info_index_, send_config_info_index_);
// ...
```
So just send the device info once, but always send the config info. Then you don't have to mess around with the `-1` everywhere.

Reply:
That's better. I don't know why I didn't think of that.